### ISSUE TITLE
Increase limit of comparison stars a user may select

### ISSUE THREAD

**Issue by ivenzor** [rzellem/EXOTIC#1337]
There is a hardcoded limit of 10 comparison stars that a user may select. If there are more than 10, the user must re-enter the comparison stars. We could change the limit to 20. If a user takes the time to manually choose 10 or more comparison stars, the additional runtime likely wouldn’t be a significant issue.

The current limit is hardcoded in `inputs.py` within the `comparison_star_coords` function:
```
def comparison_star_coords(comp_stars, rt_bool):
    if isinstance(comp_stars, list) and 1 <= len(comp_stars) <= 10 and \
            all(isinstance(star, list) for star in comp_stars):
    ...
    else:
        comp_stars = []

    if not comp_stars:
        while True:
            if not rt_bool:
                num_comp_stars = user_input("\nHow many Comparison Stars would you like to use? (1-10): ", type_=int)
                ...

```

**Comment by tamimfatahi** [comment]
Let's hold off on increasing the limit. I recommend we keep it at 10 since too many comparison stars can increase the runtime exponentially. Maybe we ignore the limit when adding AAVSO comp star.

**Comment by tamimfatahi** [comment]
Once Issue #1316 is merged into develop, please implement the suggestion above. I don't want you working on a solution when the code is being changed.

**Comment by tamimfatahi** [comment]
@ivenzor You can begin working on this issue if you'd like. Please use the `inputs.py` file in PR #1337 as reference.
